src/Spago/Version.hs

```diff
@@ -55,18 +67,18 @@ getCurrentVersion = do
 
 
 -- | Get the next version to use, or die if this would result in the version number going down/not changing.
-getNextVersion :: Spago m => VersionBump -> SemVer -> m SemVer
+getNextVersion :: VersionBump -> SemVer -> Either Text SemVer
```

Comment:
Made this pure so it's easier to unit test..

Reply:
Seems reasonable. It is possibly worth re-wording the comment so it doesn't mention `die`.

Reply:
Good point. I actually went for deleting this comment, as I find the code self-explanatory enough.